ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/IntegratedWindowsAuthentication/IntegratedWindowsAuthentication.qml

```diff
@@ -0,0 +1,156 @@
+// [WriteFile Name=IntegratedWindowsAuthentication, Category=CloudAndPortal]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+import Esri.Samples 1.0
+import Esri.ArcGISRuntime.Toolkit.Dialogs 100.6
+
+Item {
+    readonly property url arcgis_url: "http://www.arcgis.com"
+
+    MapView {
+        id: view
+        anchors.fill: parent
+    }
+
+    Rectangle {
+        anchors {
+            margins: 5
+            left: parent.left
+            top: parent.top
+        }
+        width: childrenRect.width
+        height: childrenRect.height
+        color: "#000000"
+        opacity: .70
+        radius: 5
+
+        ColumnLayout {
+            TextField {
+                id: securePortalUrl
+                Layout.fillWidth: true
+                Layout.margins: 2
+                placeholderText: qsTr("Enter portal url secured by IWA")
+                background: Rectangle {
+                    color: "white"
+                }
+            }
+
+            Row {
+                Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+                CheckBox {
+                    id: forceLoginBox
+                    checked: false
```

Comment:
should try and just use the text property on Checkbox instead of creating a Row with a Checkbox and Text separate